src/main/java/com/meilisearch/sdk/Settings.java

```diff
@@ -1,66 +1,55 @@
 package com.meilisearch.sdk;
 
+import java.util.HashMap;
 import lombok.Getter;
 import lombok.Setter;
 import org.json.JSONObject;
 
-import java.util.HashMap;
-
 /**
  * Data Structure for the Settings in an {@link Index}
  *
- * Refer https://docs.meilisearch.com/reference/api/settings.html
+ * <p>Refer https://docs.meilisearch.com/reference/api/settings.html
```

Comment:
Why sometimes the Refer is at the end of the line and sometimes it's at the beginning of the next line?